CONTRIBUTING.md

```diff
@@ -1,21 +1,33 @@
 # Contributing
 
-At Medic Mobile we welcome and appreciate community contributions.
+The Core Framework of the Community Health Toolkit is powered by people like you. Your contributions help us create open source technology for a new model of healthcare that reaches everyone.
 
-## Ways to contribute
+## [Ways to contribute](#ways-to-contribute)
+  - [First time contributor? Start here!](#first-time-contributor)
+  - [Submitting code](#submitting-code)
+  - [Disclosing vulnerabilities](#disclosing-vulnerabilities)
+  - [Improving our documentation](#improving-our-documentation)
+
+## Communication
+We recommend you raise an issue on Github or start a conversation on our [Community Forum](https://communityhealthtoolkit.org/forum) about the change you want to make before you start on code. Our Community Forum is especially helpful is you are a new contributor or find yourself unsure how to move forward with an issue.
 
-### Communication
+## Ways to contribute
 
-If you have an idea or a question we'd love to hear from you! The easiest way to get in touch is by raising issues in the [medic Github repo](https://github.com/medic/medic/issues). You can also reach out on our community Slack channel. You can request access via [this _Community Health Toolkit_ page](https://communityhealthtoolkit.org/slack).
+### First time contributor?
+Issues labeled help wanted are a great place to start. Looking for other ways to help? You can also:
+* Improve our [documentation](#improving-our-documentation)
+* Find and mark duplicate issues
+* Try to reproduce issues and help with troubleshooting
+* Or [share a new idea](https://communityhealthtoolkit.org/forum) or [question](https://communityhealthtoolkit.org/forum) with us!
```

Comment:
It's weird that both of these links are to the same place - maybe replace them with a single link so it's clear?

Reply:
Good catch!